documentation/extensions/index.md

```diff
@@ -85,6 +85,7 @@ Experimental Smack Extensions and currently supported XEPs of smack-experimental
 | [Internet of Things - Discovery](iot.md)    | [XEP-0347](http://xmpp.org/extensions/xep-0347.html) | Describes how Things can be installed and discovered by their owners. |
 | Google GCM JSON payload                     | n/a                                                  | Semantically the same as XEP-0335: JSON Containers |
 | Client State Indication                     | [XEP-0352](http://xmpp.org/extensions/xep-0352.html) | A way for the client to indicate its active/inactive state. |
+| Multi-User Chat Light                       | [XEP-xxxx]( https://s3.amazonaws.com/uploads.hipchat.com/15025/1268415/d9ByVbihpLmzxBI/XEP-xxxx.Multi.User.Chat.Light.pdf) | Multi-User Chats for mobile XMPP applications and specific enviroment. |
```

Comment:
Can't this point to the XSF's `inbox/`? Or some other official place where MUC light is provided?